versions.sh

```diff
@@ -86,12 +86,39 @@ for version in "${versions[@]}"; do
 		jq <<<"$json" -c --argjson doc "$doc" '
 			.[env.version] = {
 				version: env.fullVersion,
-				variants: [ "apache-buster", "fpm-buster", "fpm-alpine3.12" ],
+				variants: [
+					# arrays of arrays to control ordering
+					if [ "7", "8.9", "9.1" ] | index(env.version) then
+						[
+							[ "buster" ],
+							[ "alpine3.13" ]
```

Comment:
Do these ones fail on 3.14?

Reply:
Probably not. I left out `8.9` + `9.1` since they are EOL by the end of the year and `7` just got lumped in even though its EOL is at the end of 2022.